### PR TITLE
Added Geeetech Prusa i3 support

### PR DESCRIPTION
**Pull request by martin-key**
I have added support for Geeetech Prusa i3 pro B with single extruder. Firmware seems to work fine. I have made over 10 prints.

### CODE REVIEW

File: Marlin/Conditionals.h [hunk 1]
```diff
@@ -209,10 +209,10 @@
   #else //no panel but just LCD
     #if ENABLED(ULTRA_LCD)
       #ifndef LCD_WIDTH
-        #define LCD_WIDTH 16
+        #define LCD_WIDTH 20
       #endif
       #ifndef LCD_HEIGHT
-        #define LCD_HEIGHT 2
+        #define LCD_HEIGHT 3
       #endif
```
Comment (Blue-Marlin) [inline Marlin/Conditionals.h:216]: Don't change the defaults for all boards what have this undefined.
Define it in your config. Or make special case above.

3 ?

**Comment by Blue-Marlin** [comment]
A new entry in boards.h is needed.
The target branch should be RCBugFix.

**Comment by martin-key** [comment]
Hello !

I can close the request, make the changes and open another? 

Best regards,
Martin K

**Comment by CONSULitAS** [comment]
@StarbuckBG 

> I can close the request, make the changes and open another?

Yes. And perhaps you should mention this PR with a #3969 in your new text.

Thanks for your contribution!

**Pull request closed without merging**